modules/azure-gaming/game-dev-vmss/main.bicep

```diff
@@ -118,13 +118,13 @@ param parsecUserEmail string = ''
 param parsecIsGuestAccess bool = false
 
 @description('Virtual Network Resource Name')
-param vnetName string = uniqueString('vnet', vmssName, newGuid())
+param vnetName string = uniqueString(newGuid())
```

Comment:
newGuid() is going to be unique every time you deploy, so if you deploy this template twice you will end up deploying an entirely different resource. I assume this is not the behavior you are looking for?

Reply:
I keep running into this error, which I was trying to resolve this approach. But now I am not even sure which resource is getting stuck...

`Unable to edit or replace deployment 'p3qxqfi6ccpbq': previous deployment from '8/3/2022 7:48:21 PM' is still active (expiration time is '8/10/2022 7:48:18 PM').`

Reply:
okay, i confirmed its the subnet...